editor/editor_properties.cpp

```diff
@@ -3092,7 +3118,12 @@ bool EditorPropertyNodePath::is_drop_valid(const Dictionary &p_drag_data) const
 }
 
 void EditorPropertyNodePath::update_property() {
-	NodePath p = get_edited_object()->get(get_edited_property());
+	NodePath p;
```

Comment:
```suggestion
	NodePath p = get_edited_object()->get(pointer_mode ? _get_meta_pointer_property() : get_edited_property());
```
Maybe not more readable, idk

Reply:
I think its as good as it is, its more readable.